db/data/20240812102718_migrate_users_to_users_organizations.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class MigrateUsersToUsersOrganizations < ActiveRecord::Migration[6.1]
+  def up
+    User.admin(false).find_each do |user|
```

Comment:
I was just wondering about L52 in User.rb. With that in place I assume that there are situations where an admin also has the organization_id set. We should decide if we either remove that admin scope there or include the admins in the migration as well.